doc/howto/snap.md

```diff
@@ -18,7 +18,7 @@ When running LXD in a production environment, you must make sure to have a suita
 Snaps come with different channels that define which release of a snap is installed and tracked for updates.
 See [Channels and tracks](https://snapcraft.io/docs/channels) in the snap documentation for detailed information.
 
-Feature releases of LXD are available on the `latest` track.
+Feature releases of LXD are available on the `latest` track as well as numbered tracks like `6` and `6.1`.
```

Comment:
We wont be doing the 6.x tracks anymore

Reply:
I welcome that simplification!

Reply:
What's the plan with `6.1/*`? Remove it when 6.2 or 6.3 lands in `6/stable`?